frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1149,153 +1531,84 @@ function ReaderFooter:addToMainMenu(menu_items)
                 end
             },
             {
-                text = _("Show footer separator"),
+                text = _("Hide empty items"),
+                help_text = _([[This option will hide values like 0 or off.]]),
+                enabled_func = function()
+                    return self.settings.all_at_once == true
+                end,
                 checked_func = function()
-                    return self.settings.bottom_horizontal_separator == true
+                    return self.settings.hide_empty_generators == true
                 end,
                 callback = function()
-                    self.settings.bottom_horizontal_separator = not self.settings.bottom_horizontal_separator
+                    self.settings.hide_empty_generators = not self.settings.hide_empty_generators
                     self:refreshFooter(true, true)
                 end,
             },
             {
-                text = _("Lock status bar"),
-                checked_func = function()
-                    return self.settings.lock_tap == true
-                end,
-                callback = function()
-                    self.settings.lock_tap = not self.settings.lock_tap
+                text = _("Include current page in pages left"),
+                help_text = _("By default, KOReader does not include the current page when calculating pages left. For example, in a book or chapter with n pages the 'pages left' item will range from 'n-1' to 0 (last page). With this feature activated, the current page is factored in, resulting in the count going from n to 1 instead."),
```

Comment:
Current for reference.
```
_([[
Normally, the current page is not counted as remaining, so "pages left" (in a book or chapter with n pages) will run from n-1 to 0 on the last page.
With this enabled, the current page is included, so the count goes from n to 1 instead.]]),
```

Ergo,
```suggestion
                help_text = _([[
By default, KOReader does not include the current page when calculating pages left. For example, in a book or chapter with n pages the 'pages left' item will range from 'n−1' to 0 (last page).
With this feature activated, the current page is factored in, resulting in the count going from n to 1 instead.]]),
```

(focus on the actual text will come later)

Reply:
I mean my time is limited.